corehq/apps/userreports/sql.py

```diff
@@ -101,3 +104,33 @@ def _hash(domain, table_id):
 
     return 'config_report_{0}_{1}_{2}'.format(domain, table_id, _hash(domain, table_id))
 
+
+def get_expanded_columns(table_name, column_config):
+
+    session = Session()
+    connection = session.connection()
+    metadata = sqlalchemy.MetaData()
+    metadata.reflect(bind=connection)
```

Comment:
Thanks for pointing that out. I was struggling to figure out how to get the table. Should have looked more closely at the file I was editing :)